### PR TITLE
Various documentation improvements before 1.0b10

### LINKED ISSUE #4607: Add ability to open custom notebook in Panelite.
The Jupyter lite extension https://github.com/jupyterlab-contrib/jupyterlab-open-url-parameter enables you to open a notebook from an url in Jupyterlite.

This would be really powerful to include in the build of Panelite as we can the start to share links to notebooks that opens quickly for the user with a working environment.

### PR DESCRIPTION
**Pull request by philippjfr**
- Add jupyterlite-open-url extension (fixes #4607)

### CODE REVIEW

File: lite/requirements.txt [hunk 1]
```diff
@@ -7,3 +7,4 @@ jupyterlite
 nbformat
 pkginfo
 pyviz_comms
+jupyterlite-open-url-parameter
```
Comment (jtpio) [inline lite/requirements.txt:10]: Looks like this should be `jupyterlab-open-url-parameter` here.

Comment (philippjfr) [reply lite/requirements.txt:10]: Indeed, good catch thanks.

Comment (Hoxbro) [reply lite/requirements.txt:10]: I assume you followed this: https://github.com/jupyterlite/jupyterlite/blob/76cd0496add7494bec9a749b2044e28074f60781/docs/howto/content/open-url-parameter.md.

@jtpio Does that mean that the document should be updated?

Comment (jtpio) [reply lite/requirements.txt:10]: Probably this link points to an old version of the docs?

The docs look good here: https://jupyterlite.readthedocs.io/en/latest/howto/content/open-url-parameter.html

**Pull request merged**